### ISSUE TITLE
🐛 Bug Report: Cannot create indexes with name "order"

### ISSUE THREAD

**Issue by mateusz-bak** [appwrite/appwrite#2683]
### 👟 Reproduction steps

Create an index in a database with name "order".
Click save.

### 👍 Expected behavior

Index is created succesfully

### 👎 Actual Behavior

Index fails to generate,  `appwrite-worker-database` logs show:

```
SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'order ON `appwrite`.`_project_6191464048fd7_collection_61ef21443d7c227645f0` ...' at line 1
```

### 🎲 Appwrite version

Version 0.12.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by Meldiron** [comment]
I was able to reproduce the problem and I will be working towards resolving this issue.

**Comment by Meldiron** [comment]
The issue is related to utopia-php database, and query builder in `getSQLIndex` of `MariaDB` adapter. I have created PR to resolve this issue: https://github.com/utopia-php/database/pull/103

I was able to create `order` key using changes from PR above:

![CleanShot 2022-01-25 at 12 36 58](https://user-images.githubusercontent.com/19310830/150970061-f1c331b3-0e0c-4fee-a2ab-dc5a38238da3.png)

**Comment by Meldiron** [comment]
PR has been merged.

@mateusz-bak  If you are using master branch, you can simply re-build Appwrite to fetch latest changes of libraries, and get fix for this issue: `docker-compose up -d --build --force-recreate`

**Comment by TorstenDittmann** [comment]
We just released 0.13 and this should be fixed 👍🏻 

You can find the announcement [here](https://dev.to/appwrite/announcing-appwrite-013-with-major-upgrades-to-storage-and-functions-3hpf).

Thank you a lot for reporting this 🙏🏻
